src/aco/builder.rs

```diff
@@ -149,6 +149,17 @@ where
     self
   }
 
+  /// Sets the termination condition.
+  ///
+  /// For more info see [aco::termination_condition] module.
+  ///
+  /// ## Arguments
+  /// * `goodness` - Implementation of [TerminationCondition] trait.
```

Comment:
For some reason I can not suggest changes 🤷🏻 

So `goodness` -> `termination_condition`